File: web.py
import requests
from bs4 import BeautifulSoup

url = 'http://example.com'
response = requests.get(url)
soup = BeautifulSoup(response.content, 'html.parser')

# Example: Extract the first <h1> tag
h1_tag = soup.find('h1').text
print(h1_tag)

File: web.js
const axios = require('axios');
const cheerio = require('cheerio');

const url = 'http://example.com';

axios.get(url)
  .then(response => {
    const html = response.data;
    const $ = cheerio.load(html);

    // Example: Extract the first <h1> tag
    const h1Tag = $('h1').first().text();
    console.log(h1Tag);
  })
  .catch(error => console.error(error));
